src/cryptoadvance/specter/api/rest/livereadyprobes.py

```diff
@@ -0,0 +1,39 @@
+"""
+REST API Resource Routing
+
+http://flask-restful.readthedocs.io/en/latest/
+"""
+import logging
+import time
+from flask import current_app as app
+from cryptoadvance.specter.api.rest.base import BaseResource, rest_resource
+
+
+logger = logging.getLogger(__name__)
+
+
+@rest_resource
+class ResourceLiveness(BaseResource):
+    """/api/healthz/liveness"""
+
+    endpoints = ["/healthz/liveness"]
+
+    def get(self):
+        return "i am alive"
```

Comment:
If we return jsons is all get requests maybe makes sense to return json here as well, for example
`{"status": "i am alive"}`